Server/src/services/tools/blender_bridge.py

```
@@ -0,0 +1,108 @@
+    ctx: Context,
+    action: Annotated[
+        Literal["status", "scene_info", "object_info", "screenshot", "run_python",
+                "import_model", "check_updates", "sync_addon"],
+        "Operation to perform.",
+    ],
+    object_name: Annotated[str, "object_info: name of the Blender object to inspect."] | None = None,
+    object_names: Annotated[
+        list[str], "import_model: Blender objects to export (children included). Omit for selection_only or the whole scene."
+    ] | None = None,
+    selection_only: Annotated[bool, "import_model: export only what is currently selected in Blender."] | None = None,
+    format: Annotated[Literal["glb", "fbx"], "import_model: export format (default glb)."] | None = None,
```

Ruff (0.16.3)

[error] Function argument `format` is shadowing a Python builtin

(A002)